articles/azure-large-instances/workloads/epic/configure-azure-service-health-alerts.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Configure Azure service health alerta
+description: Explains how to configure Azure service health alerts.
+ms.topic: conceptual
+author: jjaygbay1
+ms.author: jacobjaygbay
+ms.service: baremetal-infrastructure
+ms.date: 06/01/2023
+---
+# Configure Azure Service Health alerts
+
+You can get automatic notifications when there are planned maintenance events or unplanned
+downtime that affects your infrastructure.
+
+Follow these steps to ensure you have Service Health alerts configured:
+
+1. Go to the [Microsoft Azure portal](https://portal.Azure.Com).
+2. Search for “service health” in the search bar and select **Service Health** from the results.
```

Comment:
@ianmsgit : Are the images from the WElcome package for this section in the process of being ported?

Reply:
I was waiting to see if you were able to get the source files - is that a definite no?

Reply:
@suzizuber : Please use the images from the Welcome package.

Reply:
@ryellajo12 I'm unfamiliar with what the Welcome package is. Is it a Word doc?